switch/main.go

```diff
@@ -0,0 +1,24 @@
+package main
+
+import "fmt"
+
+func main() {
+	i := 2
+	fmt.Print("Write", i, "as")
+	switch i {
+	case 1:
+		fmt.Println("one")
+	case 2:
+		fmt.Println("two")
+	case 3:
+		fmt.Println("three")
+	}
+	a := "BMW"
+	fmt.Print(a, " is ")
+	switch a {
+	case a:
+		fmt.Println("Car")
+	case a:
+		fmt.Println("Fruit")
```

Comment:
This is not the correct use of switch case, as you are using the same `case a` twice. But that too is incorrect as it's a variable and not a constant value. cases in switch cases mostly revolve around constants so try using that.

What I mean is you cannot use a variable in case definition